packages/wonder-blocks-dropdown/components/action-menu.md

```diff
@@ -62,6 +62,7 @@ class HybridMenu extends React.Component {
         this.state = {
             selectedValues: ["homework"],
         };
+        this.handleChange = this.handleChange.bind(this);
```

Comment:
Alternatively, and I think we may prefer this, you could do:

```
handleChange = (update) => {
  ...
}
```

in the method definition.

Reply:
Unfortunately Styleguidish doesn't support arrow functions in class field properties -- I've added comments everywhere to explain the binding in the constructor.